app/scraper/loot/scraper.py

```diff
@@ -36,10 +42,34 @@ def get_source() -> Source:
     def get_duration() -> OfferDuration:
         raise NotImplementedError("Please implement this method")
 
+    @staticmethod
+    def read_offers_from_page(driver: WebDriver) -> list[Offer]:
+        raise NotImplementedError("Please implement this method")
+
     @staticmethod
     def get_max_wait_seconds() -> int:
         return MAX_WAIT_SECONDS
 
+    @staticmethod
+    def categorize_offers(offers: list[Offer]) -> list[Offer]:
+        for offer in offers:
+
+            if re.search("(demo|trial)", offer.title[-6:], re.IGNORECASE):

```

Comment:
Added a method, but removed "trial". The disadvantage of missing e.g. "Trial of Fire" or "Fire Trial" is bigger than not catching some trial version to me.

The new "demo"-filter is a more broad but should still catch nearly no legitimate games (still catches Cha-demo) or "demo-man". I don't think we should spend too much time on that.

When demos get written to the DB instead of being discarded (see https://github.com/eikowagenknecht/lootscraper/pull/79#discussion_r919239972) we can skim the list at some point and refine the filter if need be.